tests/ecosystem/upgrade/conftest.py

```diff
@@ -201,6 +204,30 @@ def fio_conf_block():
         """)
 
 
+@pytest.fixture(scope='session')
+def fio_conf_mcg(mcg_obj_session, bucket_factory_session):
+    """
+    Basic fio configuration for upgrade utilization for AWS S3 bucket.
```

Comment:
From what I understand, the FIO configuration is for a NooBaa bucket, and not for an AWS S3 bucket - no?
I see you create the workload buckets by using `bucket_factory_session()` and also use the `mcg_obj_session.s3_endpoint`, which is the S3 endpoint of NooBaa - not AWS

Reply:
Changed